playgrounds/vite/src/Liquidity/index.tsx

```diff
@@ -79,12 +79,27 @@ export default function Liquidity({
   }, [nativeAssetValue, otherAssetValue, pluginMode]);
 
   const handleWithdraw = useCallback(async () => {
-    const ethAddress = await skClient?.thorchain.savings({
-      assetValue: nativeAsset!,
-      percent: withdrawPercent,
-      type: 'withdraw'
-    })
-    console.log("🚀 ~ handleWithdraw ~ ethAddress:", ethAddress)
+    let tx ;
+    if (pluginMode == "mayaplugin") {
+      tx = await skClient?.mayachain.withdraw({
+          assetValue: nativeAsset!,
+          percent: withdrawPercent,
+          from: "sym",
+          to : "cacao"
+      })
+      if(tx)
+        setWithdrawTx(tx);
+    }
+    else {
```

Comment:
you dont need else here - just return early

Reply:
Fixed